### PR TITLE
Temp Fix

### PR DESCRIPTION
**Pull request by ghost**
Tested

### CODE REVIEW

**Comment by ghost** [comment]
Just fixes HTTPS mixed content temp.

**Review by AvgZing — APPROVED** [review]
Looks good to me!

**Comment by AvgZing** [comment]
I left Taki out of reviewers because hes inactive atm

**Review by Ad5001 — APPROVED** [review]
Looks also good to me !

File: index.html [hunk 1]
```diff
@@ -186,7 +186,7 @@ <h4 style="margin-left: 20px;">Staff Members</h4>
           <h4>HimbeersaftLP - Admin Member</h4>
           <p>An amazing and hilarious developer who always contributes!</p>
           <ul>
-            <li><a href="https://goo.gl/9M3Xzs"><i class="fa fa-globe"></i></a></li>
+            <li><a href="https://himbeer.me"><i class="fa fa-globe"></i></a></li>
```
Comment (HimbeersaftLP) [inline index.html]: That was the first thing I wanted to do after I saw the first commit, but luckily you were faster lol thanks

**Review by HimbeersaftLP — APPROVED** [review]
Hmm, I don't think https is needed fpr the links too, but if your browser complains that let it be lol

**Comment by Ad5001** [comment]
@HimbeersaftLP No it's not needed, but it's better.

**Comment by ghost** [comment]
I will remember that for when I do the rest

**Comment by dragonwocky** [comment]
Something that is a basic fix like this doesn't need everyone to approve before you push. Once one or two people have said that it's fine then just push the commit. We don't always need to have the whole team review it (some things we should, but something small like this? Nah). 😛

**Comment by AvgZing** [comment]
:)

**Pull request merged**